testsuite/features/profiles/cloud_aws/Docker/serverhost/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Container used to test Content Management feature
+#
+# VERSION               1.0.0
+
+FROM ip-172-16-1-30.eu-central-1.compute.internal/systemsmanagement/uyuni/master/docker/containers/uyuni-master-testsuite
+MAINTAINER Michael Calmer "mc@suse.com"
```

Comment:
same as before deprecated, also notice here that you use your email again, so the first one should be the same